src/__tests__/select-options.js

```diff
@@ -133,3 +133,27 @@ test('does not select anything if options are disabled', () => {
   expect(o2.selected).toBe(false)
   expect(o3.selected).toBe(false)
 })
+
+test('should call onChange/input bubbling up the event when a new option is selected', () => {
+  const {select, form} = setupSelect({multiple: true})
+  const onChangeSelect = jest.fn()
+  const onChangeForm = jest.fn()
+  const onInputSelect = jest.fn()
+  const onInputForm = jest.fn()
+  select.addEventListener('change', onChangeSelect)
```

Comment:
The test utils exposes an addEventListener helper that should be used instead (to avoid memory leaks by benefiting from automatic cleanups).